Automattic/jetpack
# MU WPCOM: Load built version wpcom-sidebar-notice.js

## Proposed changes:

Somehow, I think https://github.com/Automattic/jetpack/pull/38233 broke `wpcom-sidebar-notice` feature. But this is weird because I swear this was not broken when I tested that...

Anyway, this PR should bring it back.

### Other information:


## Jetpack product discussion



## Does this pull request change what data or activity we track or use?



## Testing instructions:






1. Go to wp-admin of an Atomic Classic site without a mapped domain.
2. Verify you can see the sidebar upsell:

<img width="166" alt="image" src="https://github.com/user-attachments/assets/0b718ae4-60f2-4c81-8b8e-e602f80f8a77">
